Microsoft.Toolkit.Uwp.Services/Services/Storage/ILocalStorageService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Toolkit.Uwp.Services.Services.Storage
+{
+    /// <summary>
+    /// Store data in the Local environment (only on the current device)
+    /// </summary>
+    public interface ILocalStorageService : IStorageService
```

Comment:
Sure but as this is a general framework, we do not want to add more things NGOs to maintain with no obvious reasons. Here an empty interface does not make sense in the context of the toolkit